prow/lib.sh

```diff
@@ -197,13 +197,16 @@ function setup_kind_clusters() {
 
   KUBECONFIG_DIR="$(mktemp -d)"
 
+  docker network create kind
```

Comment:
nit: can we only create this if not present in case people run locally and already have this?